catalog/USGS/USGS_NLCD_RELEASES_2023_REL_RCMAP_V6_COVER_preview.js

```diff
@@ -0,0 +1,57 @@
+var dataset = ee.ImageCollection('USGS/NLCD_RELEASES/2023_REL/RCMAP/V6/COVER');

```

Comment:
this file should be in the examples/USGS directory, next to the other .js file

Reply:
Moved to the right area, thanks